packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -124,8 +124,19 @@ export function normalizeError(
 	// Anywhere they are set should be on a valid Fluid Error that would have been returned above,
 	// but we can't prove it with the types, so adding this defensive measure.
 	if (typeof error === "object" && error !== null) {
-		const { canRetry, retryAfterSeconds } = error as any;
-		Object.assign(normalizeError, { canRetry, retryAfterSeconds });
```

Comment:
the existing tests are overcomplicated and don't like empty properties, so just making the code only set properties if they exist